kratos/includes/constitutive_law.h

```diff
@@ -23,6 +23,7 @@
 
 /* Project includes */
 #include "includes/define.h"
+#include "includes/define_deprecated.h"
```

Comment:
hm I am not sure if this will still require a recompilation of pretty much everything since this includes `kratos_version.h` :/

I just tried, committing changes `kratos_version.h` and hence requires re-linking of the core

The changes of the git-version are in the cpp though, thats why I am not sure 
@loumalouomega could you please try?

@roigcarlo do you have a comment?

Reply:
The kratos_version now has a cpp, that is the refactoring that @roigcarlo did